transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestServerVerticle.java

```diff
@@ -253,10 +253,22 @@ private HttpServerOptions createDefaultHttpServerOptions() {
     serverOptions.setIdleTimeout(TransportConfig.getConnectionIdleTimeoutInSeconds());
     serverOptions.setCompressionSupported(TransportConfig.getCompressed());
     serverOptions.setMaxHeaderSize(TransportConfig.getMaxHeaderSize());
+    serverOptions.setMaxFormAttributeSize(TransportConfig.getFormAttributeSize());
+    serverOptions.setCompressionLevel(TransportConfig.getCompressionLevel());
+    serverOptions.setMaxChunkSize(TransportConfig.getMaxChunkSize());
+    serverOptions.setDecompressionSupported(TransportConfig.getDecompression());
```

Comment:
maybe `getDecompressionSupported` method name is better.

Reply:
ok